Services/Common/OJS.Services.Common.Models/Cache/ContestCategoryTreeViewModel.cs

```diff
@@ -22,13 +22,12 @@ public class ContestCategoryTreeViewModel : IMapExplicitly
     public IEnumerable<ContestCategoryTreeViewModel> Children { get; set; }
         = Enumerable.Empty<ContestCategoryTreeViewModel>();
 
+    public IEnumerable<AllowedContestStrategiesServiceModel> AllowedStrategyTypes { get; set; }
+        = Enumerable.Empty<AllowedContestStrategiesServiceModel>();
     public void RegisterMappings(IProfileExpression configuration)
         => configuration
             .CreateMap<ContestCategory, ContestCategoryTreeViewModel>()
             .ForMember(
```

Comment:
Remove AllowedStrategyTypes from ContestCategoryTreeViewModel if you dont need to map it

Reply:
I am using in ContestCategoriesBusiness logic -> row 112-119